src/Microsoft.AspNetCore.Server.Kestrel/Infrastructure/Constants.cs

```diff
@@ -20,5 +23,20 @@ internal class Constants
         /// for info on the format.
         /// </summary>
         public const string RFC1123DateFormat = "r";
+
+        private static int GetECONNRESET()
+        {
+            switch (PlatformServices.Default.Runtime.OperatingSystemPlatform)
+            {
+                case Platform.Windows:
+                    return -4077;
+                case Platform.Linux:
+                    return -104;
+                case Platform.Darwin:
+                    return -54;
+                default:
+                    throw new Exception("Unable to determine ECONNRESET value on this platform.");
```

Comment:
I'm not sure this should be throwing at all. Maybe we just log that we couldn't determine the ECONNRESET constants.

This really only affects the noisiness of the logs, and it is possible that that someone could bring their own libuv.